layouts/page/ecosystem.html

```diff
@@ -390,19 +390,19 @@ <h2><i class="fa-regular fa-gauge-circle-bolt"></i> Triton </h2>
               class="fa-solid fa-arrow-up-right"></i> </a>
         </div>
         <div class="col-md-6 py-3">
-          <h2><i class="fa-regular fa-gauge-circle-bolt"></i> Merlin </h2>
+          <h2><i class="fa-regular fa-gauge-circle-bolt"></i> NeMo Curator </h2>
           <div class="tags-container border-top border-2 border-rapids-purple-primary">
             <span class="tags-item text-rapids-purple-primary border-rapids-purple-primary"><i
-                class="fa-solid fa-tag"></i> RECOMMENDERS </span>
+              class="fa-solid fa-tag"></i> CURATION </span>
             <span class="tags-item text-rapids-purple-primary border-rapids-purple-primary"><i
-                class="fa-solid fa-tag"></i> NVIDIA AI </span>
+                class="fa-solid fa-tag"></i> PYTHON </span>
+            <span class="tags-item text-rapids-purple-primary border-rapids-purple-primary"><i
+                class="fa-solid fa-tag"></i> NEMO </span>
```

Comment:
What do these tags really mean? We only have one tag for "NEMO". Would it be better to have the tag "NVIDIA AI"?

Reply:
They are more closely aligned with NeMo projects vs NVAIE